lib/recurly/validate.js

```diff
@@ -19,7 +19,14 @@ const TYPES = [
   },
   {
     type: 'master',
-    pattern: /^5[0-5]/,
+    pattern: {
+      test: (n) => {
+        let firstSix = parseInt(n.substr(0, 6), 10);
+        if (/^5[0-5]/.test(n)) return true;
```

Comment:
This can be tightened just a little... MasterCards do not start with "50", they can start with "5[1-5]".